### ISSUE TITLE
Error in Google order style (I100)

### ISSUE THREAD

**Issue by pferate** [PyCQA/flake8-import-order#80]
A change in the 0.9 release broke the Google order style, at least for `I100`.  It looks like it is not ignoring the case of letters anymore.

According to the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting)
> Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module's full package path.

Here is an example that worked in 0.8, but is flagging `I100` in 0.9:
```python
import os
import StringIO
import sys
```

This (incorrect) example passes in 0.9:
```python
import StringIO
import os
import sys
```

**Comment by pgjones** [comment]
Thanks, will fix shortly.

**Comment by pgjones** [comment]
Should be fixed by 3f371dded0b19dba24e0287786fc615cfe3624cd, can you confirm?

**Comment by pferate** [comment]
Looks like that works.  Thanks!

**Comment by pgjones** [comment]
Thanks, released 0.9.1.

**Comment by pferate** [comment]
Thanks!  Works great.

**Comment by wjwwood** [comment]
I think this might not be fixed for `from ... import ...` statements:

```
% cat ./from_import_order_bug.py
from foo import bar
from foo import Baz

% flake8 - < ./from_import_order_bug.py
stdin:1:1: F401 'foo.bar' imported but unused
stdin:2:1: I100 Import statements are in the wrong order. from foo should be before from foo
stdin:2:1: F401 'foo.Baz' imported but unused
```

I'm using Python 3.5:

```
% python3 -m pip freeze | grep flake8
flake8==3.0.4
flake8-import-order==0.10
```

I believe my colleague @dirk-thomas might be looking into a patch.

**Comment by wjwwood** [comment]
https://github.com/PyCQA/flake8-import-order/pull/95

**Comment by pgjones** [comment]
The further issues should be fixed following ab53d0fa8e23de11bd572a9cb74d719076fc0ae5
